site/zenodo_rdm/serializers/codemeta.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Zenodo codemeta serializer."""
+
+from flask_resources import BaseListSchema, MarshmallowSerializer
+from flask_resources.serializers import JSONSerializer
+from invenio_rdm_records.contrib.codemeta.processors import CodemetaDumper
+from invenio_rdm_records.resources.serializers.codemeta.schema import CodemetaSchema
+from marshmallow import missing
+
+
+class ZenodoCodemetaSchema(CodemetaSchema):
+    """Zenodo Codemeta schema."""
+
+    def get_sameAs(self, obj):
+        """Get sameAs field."""
+        ret = super().get_sameAs(obj) or []
+        swhid = obj.get("swh", {}).get("swhid")
+        if swhid:
+            ret.append({"@id": swhid})
```

Comment:
For codemeta, we discussed adding the field in `identifier`. Some repositories, e.g. HAL, serializer `identifier` with multiple values (e.g. `doi`, `swhid`, etc). Since we only serializer one identifier, I added the `swhid` in the alternate identifiers with `isIdenticalTo` relationship (like we do for Datacite)